warning: possible EventEmitter memory leak detected. 11 listeners added.

After reusing 10 times a connection from the pool you get a warning like:

```
[ERROR] console - (node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit.
[2017-01-16 11:25:58.056] [ERROR] console - Trace
    at addListener (events.js:160:15)
    at once (events.js:185:8)
    at Function.queryCanceller (/path/to/sql-api/node_modules/cartodb-psql/lib/psql.js:424:24)
...
```

--- a/NEWS.md
+++ b/NEWS.md
@@ -3,6 +3,9 @@
 ## 1.43.1
 Released 2017-mm-dd
 
+Announcements:
+ * Upgrade cartodb-psql to 0.7.1.
+
 
 ## 1.43.0
 Released 2017-01-16

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
   "dependencies": {
     "bintrees": "1.0.1",
     "bunyan": "1.8.1",
-    "cartodb-psql": "0.7.0",
+    "cartodb-psql": "0.7.1",
     "cartodb-query-tables": "0.2.0",
     "cartodb-redis": "0.13.2",
     "debug": "2.2.0",

